### PR TITLE
Add tests for uint256 module

### PR DESCRIPTION
**Pull request by fmoletta**
TESTING STATUS:

**Uint256 Module**

- `uint256_add()`: FAILING (Issue with ids structs + ids constants)
-  `uint256_sqrt()`: FAILING (Issue with ids structs + ids constants)
- `uint256_unsigned_div_rem()`: FAILING (Issue with ids structs + ids constants)
- `uint256_signed_nn()`: FAILING (Issue with ids structs)
- `bigint_to_uint256()`: FAILING (Issue with ids structs + ids constants)

### CODE REVIEW

File: cairo_programs/bigint_to_uint256.cairo [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from starkware.cairo.common.cairo_secp.constants import BASE
+from starkware.cairo.common.cairo_secp.bigint import BigInt3
+from starkware.cairo.common.math_cmp import RC_BOUND
+from starkware.cairo.common.math import assert_nn_le
+
+func bigint_to_uint256{range_check_ptr}(x : BigInt3) -> (res : Uint256):
+    let low = [range_check_ptr]
+    let high = [range_check_ptr + 1]
+    let range_check_ptr = range_check_ptr + 2
+    %{ ids.low = (ids.x.d0 + ids.x.d1 * ids.BASE) & ((1 << 128) - 1) %}
```
Comment (pefontana) [inline cairo_programs/bigint_to_uint256.cairo:12]: Add the # TEST tag

**Review by pefontana — CHANGES_REQUESTED** [review]

File: cairo_programs/bigint_to_uint256.cairo (hunk 1 above)
Comment (fmoletta) [reply cairo_programs/bigint_to_uint256.cairo:12]: Thanks! I missed it

**Comment by HermanObst** [comment]
[Test common library hints UINT256](https://trello.com/c/volkcTcl/26-test-common-library-hints-uint256)

**Review by pefontana — APPROVED** [review]

File: hints_tests.py [hunk 2]
```diff
@@ -25,6 +25,11 @@ def test_program(program_name: str):
     # test_program("ec_doble")
     test_program("memcpy")
     test_program("memset")
+    #test_program("uint256_add")
+    #test_program("uint256_sqrt")
+    #test_program("uint256_unsigned_div_rem")
+    #test_program("uint256_signed_nn")
+    test_program("bigint_to_uint256")
```
Comment (Juan-M-V) [inline hints_tests.py:32]: This test throws the error `ValueError: Custom Hint Error: ValueError: Failed to get ids value`, is that expected?

**Review by Juan-M-V — APPROVED** [review]

**Pull request merged**